e2e/src/app.e2e-spec.ts

```diff
@@ -12,42 +12,30 @@ describe("workbench-client", () => {
     page.navigateTo();
   });
 
-  afterEach(async () => {
+  /**
+   * While this expectation is good, currently it will always fail because
+   * the CMS throws an error when attempting to insert styles/scripts from
+   * a different domain.
+   */
+  /* afterEach(async () => {
```

Comment:
I'd ague that's not a good assertion. There are plenty of logged errors that do not equate to a failed test.

Also, provide more detail/issue link to this CMS issue